test/util/handler-utils.ts

```diff
@@ -47,7 +47,7 @@ export const mockContext: Context = {
   logGroupName: '',
   logStreamName: '',
   getRemainingTimeInMillis: function (): number {
-    throw new Error('Function not implemented.');
+    return 0;
```

Comment:
why does this return 0?

Reply:
To disable Middy timeout during testing. See: https://middy.js.org/docs/intro/testing/